### DISCUSSION TITLE
How to set initial soil moisture condition

### DISCUSSION THREAD

**Discussion by ViniciusBMoreira** [mhm-ufz/mHM#35] [Q&A] [answered]
Hey to all!

I was readind the article _Predictions in a data-sparse region using a regionalized grid-based hydrologic model driven by remotely sensed data_ by Luis Samaniego, Rohini Kumar and Conrad Jackisch and as my goal is also to run the model with remotely sensed data only, I saw that the authors used initial soil moisture conditions. But checking the manual and configuration files, I didn't see anything about it.

Doubts:
How is this done?
Is it useful to use remotely sensed soil moisture data to set initial conditions?

ty !

Cheers 
Bryan

**Comment by rohinikumar82** [comment, marked as answer]
Hi @BryanSouzaM,

You can use the functionality of restart option and then adjust/specify the initial soil moisture values in the restart.nc file. 

Is it useful to use remotely sensed soil moisture data to set initial conditions?
- It very much depends on values you get and your research investigation. 
- Note that in the paper you mention - we did not use initial soil moisture conditions from remote sensing datasets, but I guess it was investigation to see the sensitivity of model output to initial conditions. 
- Also I guess there will be limitation of using remote sensing based SM in mHM: uncertainty and errors in RS data as well how to use them for deeper layers? 

Best,
Rohini
